xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -685,11 +693,11 @@
         <Parameter Name="advancePastDelimiter" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="sequence">To be added.</param>
-        <param name="delimiter">To be added.</param>
-        <param name="advancePastDelimiter">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="sequence">The read data, if any.</param>
+        <param name="delimiter">The delimiter to look for.</param>
+        <param name="advancePastDelimiter"><see langword="true" /> to move past the <paramref name="delimiter" /> if found.</param>
+        <summary>Tries to read everything up to the given <paramref name="delimiter" />.</summary>
+        <returns><see langword="true" /> if the <paramref name="delimiter" /> was found.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the <paramref name="delimiter" /> was found; otherwise, <see langword="false" />.</returns>
```